### PR TITLE
update doc regarding new menus organization

### PR DESCRIPTION
**Pull request by aminelouati**

### CODE REVIEW

File: src/docs/user/references/LDAPQueryTasks.adoc [hunk 1]
```diff
@@ -1,16 +1,12 @@
-=== LDAP Query Tasks
+=== LDAP Query
 
-==== Description
+The LDAP Query connector enables connection to an existing Microsoft Active Directory server to perform search operations and to check users' credentials and their membership groups.
```
Comment (mboussaa) [inline src/docs/user/references/LDAPQueryTasks.adoc]: access group is better?

File: src/docs/user/references/Tasks.adoc [hunk 2]
```diff
@@ -1,4 +1,10 @@
-This section describes the available predefined tasks in the *Tasks* menu of *ProActive Studio* and the operating systems where they run.
+This section describes the available predefined tasks in the *Tasks* menu of *ProActive Studio* and the operating systems where they run. These tasks are grouped in three sections:
+
+1. _OS Shells_: Shell, Bash, Windows Cmd, PowerShell
+2. _Languages_: Java, Scalaw, Javascript, Groovy, Ruby, Jython, Python, Perl, PHP, R
+3. _Containers_: Docker File, Docker Compose, Kubernetes
```
Comment (mboussaa) [inline src/docs/user/references/Tasks.adoc]: Docker File => Dockerfile

File: src/docs/user/references/Tasks.adoc [hunk 3]
```diff
@@ -83,24 +79,24 @@ This section describes the available predefined tasks in the *Tasks* menu of *Pr
...
 |R
 |Sample <<../user/ProActiveUserGuide.adoc#_script_tasks,Script Task>> using the `R` script engine.
 |Linux,Windows,Mac
 |R must be installed and the rJava library
 
-|Cron
-|Task executed periodically using a cron syntax
-|Linux,Mac,Windows
-|
+|Docker File
```
Comment (mboussaa) [inline src/docs/user/references/Tasks.adoc]: same

File: src/docs/user/references/LDAPQueryTasks.adoc (hunk 1 above)
Comment (aminelouati) [reply src/docs/user/references/LDAPQueryTasks.adoc]: yes it is. I will update. Thanks

File: src/docs/user/references/Tasks.adoc (hunk 2 above)
Comment (aminelouati) [reply src/docs/user/references/Tasks.adoc]: In the Tasks menu, it is rather Docker_File and Docker_Compose

Comment (mboussaa) [reply src/docs/user/references/Tasks.adoc]: Then u might use`Docker_File`

Comment (mboussaa) [reply src/docs/user/references/Tasks.adoc]: even though Docker_File is not correct and should be `Dockerfile` in the studio and in the doc

Comment (aminelouati) [reply src/docs/user/references/Tasks.adoc]: Dac

**Pull request merged**